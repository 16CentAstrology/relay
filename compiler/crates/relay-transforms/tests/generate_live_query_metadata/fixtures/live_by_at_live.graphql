query TestRootField($id: ID!) {
  node(id: $id) @live {
    id
  }
}

query TestFragmentSpread($id: ID!) {
  node(id: $id) {
    id
    ... on User @live {
      birthdate {
        day
      }
    }
  }
}

query TestInlineFragment($id: ID!) {
  node(id: $id) {
    id
    ... UserBirthdate @live
  }
}

fragment UserBirthdate on User {
  birthdate {
    day
  }
}

query TestLinkedField($id: ID!) {
  node(id: $id) {
    id
    ... on User {
      birthdate @live {
        day
      }
    }
  }
}
