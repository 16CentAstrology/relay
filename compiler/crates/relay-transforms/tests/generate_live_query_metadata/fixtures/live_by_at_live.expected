==================================== INPUT ====================================
query TestRootField($id: ID!) {
  node(id: $id) @live {
    id
  }
}

query TestFragmentSpread($id: ID!) {
  node(id: $id) {
    id
    ... on User @live {
      birthdate {
        day
      }
    }
  }
}

query TestInlineFragment($id: ID!) {
  node(id: $id) {
    id
    ... UserBirthdate @live
  }
}

fragment UserBirthdate on User {
  birthdate {
    day
  }
}

query TestLinkedField($id: ID!) {
  node(id: $id) {
    id
    ... on User {
      birthdate @live {
        day
      }
    }
  }
}
==================================== OUTPUT ===================================
query TestFragmentSpread(
  $id: ID!
) @__metadata(live: {config_id: ""}) {
  node(id: $id) {
    id
    ... on User @live {
      birthdate {
        day
      }
    }
  }
}

query TestInlineFragment(
  $id: ID!
) @__metadata(live: {config_id: ""}) {
  node(id: $id) {
    id
    ...UserBirthdate @live
  }
}

query TestLinkedField(
  $id: ID!
) @__metadata(live: {config_id: ""}) {
  node(id: $id) {
    id
    ... on User {
      birthdate @live {
        day
      }
    }
  }
}

query TestRootField(
  $id: ID!
) @__metadata(live: {config_id: ""}) {
  node(id: $id) @live {
    id
  }
}

fragment UserBirthdate on User {
  birthdate {
    day
  }
}
